--- httptunnel-3.3/Makefile.am~	Thu Jun 29 23:29:49 2000
+++ httptunnel-3.3/Makefile.am	Tue Oct 30 03:08:33 2001
@@ -5,12 +5,6 @@
 bin_PROGRAMS = htc hts
 man_MANS = hts.1 htc.1
 
-if SRCDIR
-CPPFLAGS += -I$(VPATH)/port
-else
-CPPFLAGS += -Iport
-endif
-
 htc_SOURCES = htc.c common.c tunnel.c http.c base64.c
 htc_LDADD = -Lport -lport
 hts_SOURCES = hts.c common.c tunnel.c http.c
--- httptunnel-3.3/acinclude.m4~	Fri Oct 15 19:13:45 1999
+++ httptunnel-3.3/acinclude.m4	Tue Oct 30 03:02:09 2001
@@ -1,7 +1,7 @@
 dnl HTTPTUNNEL_TYPE_SOCKLEN_T
 dnl Check for the existance of type socklen_t.
 
-AC_DEFUN(HTTPTUNNEL_TYPE_SOCKLEN_T,
+AC_DEFUN([HTTPTUNNEL_TYPE_SOCKLEN_T],
 [AC_CACHE_CHECK([for socklen_t], ac_cv_httptunnel_type_socklen_t,
 [
   AC_TRY_COMPILE(
@@ -20,7 +20,7 @@
 dnl HTTPTUNNEL_DEFINE_INADDR_NONE
 dnl Check for the existance of define INADDR_NONE
 
-AC_DEFUN(HTTPTUNNEL_DEFINE_INADDR_NONE,
+AC_DEFUN([HTTPTUNNEL_DEFINE_INADDR_NONE],
 [AC_CACHE_CHECK([whether INADDR_NONE is defined], ac_cv_httptunnel_define_inaddr_none,
 [
   AC_TRY_COMPILE(
--- httptunnel-3.3/configure.in~	Sun Feb 25 13:06:51 2001
+++ httptunnel-3.3/configure.in	Tue Oct 30 03:11:27 2001
@@ -11,14 +11,6 @@
               [  --enable-debug            enable debugging],
               AC_DEFINE(DEBUG_MODE, 1))
 
-if test "x$CFLAGS" = x; then
-	if test "x$enable_debug" != x; then
-		CFLAGS="-g -Wall -Wstrict-prototypes -Wmissing-prototypes -Wpointer-arith"
-	else
-		CFLAGS="-O2 -Wall -fomit-frame-pointer"
-	fi
-fi
-
 dnl Checks for programs.
 AC_PROG_AWK
 AC_PROG_CC
@@ -55,4 +47,4 @@
 AC_CHECK_FUNCS(socket strdup strerror daemon vsyslog)
 AC_CHECK_FUNCS(poll select endprotoent vsnprintf syslog)
 
-AC_OUTPUT(Makefile port/Makefile port/sys/Makefile)
+AC_OUTPUT(Makefile)
--- httptunnel-3.3/Makefile.am~	Tue Oct 30 03:12:03 2001
+++ httptunnel-3.3/Makefile.am	Tue Oct 30 03:13:32 2001
@@ -1,7 +1,5 @@
 ## Copyright (C) 1999 Lars Brinkhoff
 
-SUBDIRS = port
-
 bin_PROGRAMS = htc hts
 man_MANS = hts.1 htc.1
 
